src/libutil/hash.hh

```diff
@@ -44,6 +44,12 @@ struct Hash
        string. */
     Hash(std::string_view s, HashType type = htUnknown);
 
+    Hash(const Hash &) = default;
+
+    Hash(Hash &&) = default;
+
+    Hash & operator = (const Hash &) = default;
```

Comment:
Do we need these? They're already defined by default right?

Reply:
I will try without, we might have only needed them for `StorePath` when I was debugging STL coppying issues.

Reply:
Not needed!